site/content/en/docs/tasks/manage/setup_multikueue.md

```diff
@@ -130,3 +130,120 @@ It provides a robust, modular, and extensible framework that helps other open so
 
 The integration with OCM is an optional solution that enables Kueue users to streamline the MultiKueue setup process, automate the generation of MultiKueue specific Kubeconfig, and enhance multicluster scheduling capabilities.
 For more details about this solution, please refer to this [link](https://github.com/open-cluster-management-io/ocm/tree/main/solutions/kueue-admission-check).
+
+## Setup MultiKueue with ClusterProfile API
+
+{{% alert title="Note" color="primary" %}}
+This feature requires Kueue v0.15 or newer and the `v1beta2` API.
+{{% /alert %}}
+
+The [ClusterProfile API](https://multicluster.sigs.k8s.io/concepts/cluster-profile-api/) provides a standardized, vendor-neutral interface for presenting cluster information. It allows defining cluster access information in a standardized `ClusterProfile` object and using credential plugins for authentication.
+
+### Create ClusterProfile objects
+
+If you are using a cloud provider, refer to the documentation on how to generate ClusterProfile objects (e.g. [GKE](https://docs.cloud.google.com/kubernetes-engine/fleet-management/docs/generate-inventory-for-integrations)). Alternatively, you can manually create a `ClusterProfile` object for your cluster.
+
+```yaml
+apiVersion: multicluster.x-k8s.io/v1alpha1
+kind: ClusterProfile
+metadata:
+  name: ${WORKER_CLUSTER_NAME}
+  namespace: kueue-system
+spec:
+  ...
+status:
+  accessProviders:
+  - name: ${ACCESS_PROVIDER_NAME}
+    cluster:
+      server: https://${SERVER_ENDPOINT}
+      certificate-authority-data: ${CERTIFICATE_AUTHORITY_DATA}
+```
+
+### Configure Kueue Manager
+
+Next, enable the `MultiKueueClusterProfile` feature gate in your Kueue manager's configuration and define your credentials providers.
```

Comment:
I would suggest moving this information about setting the feature gate to the top, and referencing the page https://kueue.sigs.k8s.io/docs/installation/#change-the-feature-gates-configuration since we have 2 ways of enabling feature gates in Kueue. Plus, drop the FG from the configMap below.